languages/julia/maintainers.md

```diff
@@ -4,8 +4,7 @@ These awesome people help maintain the Julia track.
 
 ## Senior Maintainers
 
-TODO: add senior maintainers
+- Sascha Mann (Slack: `@Sascha Mann`, GitHub: [SaschaMann](https://github.com/SaschaMann/))
```

Comment:
You might want to sort this lexographically/alpahbetically?

Reply:
I went with chronological order (Logan joined very recently, but before the v3 push) but I'm fine with whatever order